docs/data/date-pickers/date-time-picker/DateTimePickerOpenTo.js

```diff
@@ -2,14 +2,15 @@ import * as React from 'react';
 import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
-import { MobileDateTimePicker } from '@mui/x-date-pickers/MobileDateTimePicker';
+import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+import { MobileTimePicker } from '@mui/x-date-pickers/MobileTimePicker';
 
 export default function DateTimePickerOpenTo() {
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
       <DemoContainer components={['MobileDateTimePicker', 'MobileDateTimePicker']}>
-        <MobileDateTimePicker label={'"year"'} openTo="year" />
-        <MobileDateTimePicker label={'"hours"'} openTo="hours" />
+        <DateTimePicker label={'"year"'} openTo="year" />
+        <MobileTimePicker label={'"hours"'} openTo="hours" />
```

Comment:
I can't use the responsive one here because on desktop it make no sense.
See #8989 for context

Reply:
We sort of could use it. It's just that the focus would be set on `hours`, but visually it wouldn't be super clear 🤔